obspy/core/inventory/response.py

```diff
@@ -1663,6 +1663,42 @@ def get_sacpz(self):
         paz = self.get_paz()
         return paz_to_sacpz_string(paz, self.instrument_sensitivity)
 
+    @staticmethod
+    def from_paz(zeros, poles, stage_gain,
+                 stage_gain_frequency=1.0, input_units='VOLTS',
+                 output_units='M/S', normalization_frequency=None,
+                 pz_transfer_function_type='LAPLACE (RADIANS/SECOND)',
+                 normalization_factor=1.0):
+        """
+        Takes in lists of complex poles and zeros and returns a Response with
+        those values defining its only stage.
+        :type zeros: list of complex
+        :param zeros: All zeros of the response to be defined.
+        :type poles: list of complex
+        :param poles: All poles of the response to be defined.
+        :type stage_gain: double
+        :param stage_gain: The gain value of the response [sensitivity]
+
+        Most of the optional parameters defined here are from
+        :class:`~obspy.core.inventory.response.PolesZerosResponseStage`
+        Note that if no value is specified for the normalization_frequency
+        parameter, the value assigned to stage_gain_frequency will be used.
+        :returns: new Response instance with given P-Z values
+        """
+        if normalization_frequency is None:
+            normalization_frequency = stage_gain_frequency
+        sequence = 1  # must be stage 1 because paz defines entire response
+        pzstage = PolesZerosResponseStage(
+                    sequence, stage_gain,
+                    stage_gain_frequency, input_units, output_units,
+                    pz_transfer_function_type, normalization_frequency, zeros,
+                    poles, normalization_factor)
+        sensitivity = InstrumentSensitivity(stage_gain, stage_gain_frequency,
+                                            input_units, output_units)
+        resp = Response(instrument_sensitivity=sensitivity,
+                        response_stages=[pzstage])
```

Comment:
Please add a `resp.recalculate_overall_sensitivity()` at the end to make sure the overall sensitivity is actually correct at the end.  If `stage_gain_frequency` and `normalization_frequency` differ this is not necessarily the case.

Reply:
I've made this change (added the recalculate_overall_sensitivity() call).

I would like to make sure that my test captures the desired behavior for this -- for some reason, even if stage_gain_frequency and normalization_frequency are set to the same value, I'm getting a different result for the response's sensitivity compared to the gain for the poles-zeros. This seems like this shouldn't be the case, since the PZ stage is the only one in the response. 

To be clear, when I set the stage gain to 2014943641.6, that IS the stage gain for the PZ stage. However, the overall gain looks something like 432115652.548. The frequencies for stage gain and normalization are both kept at 1.0, as is the normalization factor. Is this an issue with the units, or is there a step that I'm missing somewhere?